doc/installation_instructions/platform.rst

```diff
@@ -0,0 +1,22 @@
+.. _install-notes:
+
+Installation on specific platforms
+==================================
+
+The following describes installation details for various systems and platforms
+that SmartSim may be used on.
```

Comment:
Im not sure if this document is giving the correct format, it does not include any of the :include listed below, only the title and the single sentence

Reply:
negative